test/mermaid.jl

```diff
@@ -24,6 +24,7 @@ end # module
 @version SchemaRadV1 begin
     foo::Union{String,Missing}
     list::Vector{Int}
+    d::Val{Symbol("""\"hi1232}{{}:yj,;./[]]""")}
```

Comment:
hah, this also ruins github's syntax highlighting 😂

Reply:
I think this is https://github.com/JuliaEditorSupport/atom-language-julia/issues/219 (which appear to be used by GitHub via Linguist and VSCode)